spec/fixtures/users.yml

```diff
@@ -1,13 +1,13 @@
 DEFAULTS: &DEFAULTS
+  email: $LABEL@fixtures.org
```

Comment:
This change interpolates the fixture label into the email address, allowing us to DRY up the fixtures, as we already used that email format, i.e.:
```yaml
affiliate_admin:
  <<: *DEFAULTS
  email: affiliate_admin@fixtures.org
```

I left the email addresses with a `.gov` address, as that triggers different logic.